ktoolbox/common.py

```diff
@@ -0,0 +1,1487 @@
+import abc
```

Comment:
Move this out to be a separate project instead of copy pasting it.

Reply:
done. Now at https://github.com/thom311/ktoolbox/